content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -8,7 +8,12 @@ weight: 40
 
 You can use _topology spread constraints_ to control how {{< glossary_tooltip text="Pods" term_id="Pod" >}} are spread across your cluster among failure-domains such as regions, zones, nodes, and other user-defined topology domains. This can help to achieve high availability as well as efficient resource utilization.
 
-
+{{< note >}}
+Topology spread constraints is GA since v1.19. In prior versions, you must
+enable the `EvenPodsSpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](/docs/concepts/overview/components/#kube-apiserver) and the
+[scheduler](/docs/reference/generated/kube-scheduler/).
```

Comment:
How I'd do it:
```suggestion
{{< feature-state for_k8s_version="v1.19" state="stable" >}}
<!-- leave this shortcode in place until the note about EvenPodsSpread is obsolete -->

You can use _topology spread constraints_ to control how {{< glossary_tooltip text="Pods" term_id="Pod" >}} are spread across your cluster among failure-domains such as regions, zones, nodes, and other user-defined topology domains. This can help to achieve high availability as well as efficient resource utilization.

 {{< note >}}
In versions of Kubernetes before v1.19, you must enable the `EvenPodsSpread`
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
on the [API server](/docs/concepts/overview/components/#kube-apiserver) and the
[scheduler](/docs/reference/generated/kube-scheduler/) in order to use Pod
topology spread constraints.
```

Reply:
Done.